cirq/google/programs_test.py

```diff
@@ -259,9 +260,9 @@ def test_z_proto_dict_convert():
             }
         }
     }
-    assert_proto_dict_convert(gate, proto_dict,
-                              cirq.GridQubit(2, 3))
-
+    with pytest.raises(NotImplementedError):
```

Comment:
This should still be implemented. Do not convert this check into an expected failure, instead of an assertion of success.

What shouldn't be implemented is formulas such as `sympy.Symbol('k') + sympy.Symbol('k2')`.